src/pages/NotFoundPage/NotFoundPage.jsx

```diff
@@ -1,7 +1,15 @@
+import "./not-found-page.css";
+import notFoundImage from "../../assets/others/404-not-found-illustration.svg";
+
 const NotFoundPage = () => {
   return (
-    <div>
+    <div className="main-not-found flex-col flex-align-center flex-justify-center gap-5">
       <h1>404, Sorry this page doesn't exist!! Try later.</h1>
+      <img
```

Comment:
404 page does not have the svg you have in your assets. Take care of it.

Reply:
It's already added in src attribute. Actually, might now be showing in live app bcoz of unmerged PR. Checked again and it's showing up now. Thanks!